Library/Homebrew/rubocops/formula_desc_cop.rb

```diff
@@ -0,0 +1,78 @@
+require_relative "./extend/formula_cop"
+require_relative "../extend/string"
+
+module RuboCop
+  module Cop
+    module Homebrew
+      # This cop audits `desc` in Formulae
+      #
+      # - Checks for existence of `desc`
+      # - Checks if size of `desc` > 80
+      # - Checks if `desc` begins with an article
+      # - Checks for correct usage of `command-line` in `desc`
+      # - Checks if `desc` contains the formula name
+      class FormulaDesc < FormulaCop
+        def audit_formula(node, class_node, _parent_class_node, body)
```

Comment:
I think having `find_method_call` which always returns the `first` would be a sensible first step. If it's needed later we can have `find_method_calls` for e.g. checking use of things like `inreplace` and then loop over them.